src/NFTBill.sol

```diff
@@ -6,6 +6,10 @@ import 'openzeppelin-contracts/contracts/token/ERC20/ERC20.sol';
 
 import {IMetadata} from './interfaces/IMetadata.sol';
 
+error NotEnoughWei();
+error TooMuchEth();
+error NoCoinsSent();
```

Comment:
I think `NotEnoughWei` and `NoCoinsSent` can be unified into a single error (because conceptually it is the same error!), maybe something like `InsufficientValue` or `ZeroValue`

Reply:
Maybe:

```solidity
error ValueTooLarge();
error ValueTooSmall();
```

Reply:
Good call 👍